src/schema/v2/marketingCollections.ts

```diff
@@ -329,10 +350,12 @@ export const MarketingCollections: GraphQLFieldConfig<void, ResolverContext> = {
       category?: string
       sort?: string
     } = {
-      size,
-      artist_id: args.artistId,
-      is_featured_artist_content: args.isFeaturedArtistContent,
       ...args,
+      size,
+      ...(args.artistID && { artist_id: args.artistID }),
+      ...(args.isFeaturedArtistContent && {
+        is_featured_artist_content: args.is_featured_artist_content,
+      }),
```

Comment:
**question**: Was this additional null checking required for the feature or the get tests passing?

Reply:
It is to get test passing

Reply:
I'd recommend making the test more selective [as suggested here](https://github.com/artsy/metaphysics/pull/6187#discussion_r1822195452) rather adding these additional checks and object spreads.